src/pages/Store/index.jsx

```diff
@@ -55,7 +51,7 @@ export default class Store extends Component {
   async componentWillReceiveProps({ match, rootStore }) {
     const { params } = match;
     if (params.category) {
-      await rootStore.appStore.fetchApps({
+      await rootStore.appStore.fetchAll({
```

Comment:
The fetchAll query defalut limit 10 datas

Reply:
all right